src/DFGNode.cpp

```diff
@@ -37,6 +39,33 @@ DFGNode::DFGNode(int t_id, bool t_precisionAware, Instruction* t_inst,
   initType();
 }
 
+DFGNode::DFGNode(int t_id, DFGNode* old_node) {
```

Comment:
Add comment above? When do we need this type of copy?

Reply:
Only when we tune division patterns.